capreolus/extractor/berttext.py

```diff
@@ -28,67 +34,19 @@ class BertText(Extractor):
     config_spec = [ConfigOption("maxqlen", 4), ConfigOption("maxdoclen", 800), ConfigOption("usecache", False)]
 
     pad = 0
-    pad_tok = "<pad>"
+    pad_tok = "[PAD]"
 
     def load_state(self, qids, docids):
         with open(self.get_state_cache_file_path(qids, docids), "rb") as f:
             state_dict = pickle.load(f)
             self.qid2toks = state_dict["qid2toks"]
-            self.docid2toks = state_dict["docid2toks"]
-            self.clsidx = state_dict["clsidx"]
-            self.sepidx = state_dict["sepidx"]
 
     def cache_state(self, qids, docids):
         os.makedirs(self.get_cache_path(), exist_ok=True)
         with open(self.get_state_cache_file_path(qids, docids), "wb") as f:
-            state_dict = {"qid2toks": self.qid2toks, "docid2toks": self.docid2toks, "clsidx": self.clsidx, "sepidx": self.sepidx}
+            state_dict = {"qid2toks": self.qid2toks}
             pickle.dump(state_dict, f, protocol=-1)
 
-    def get_tf_feature_description(self):
```

Comment:
`berttext` is not only used by denserank, and hence the TF specific things can be removed. `bertpassage.py` is used for `TFKNRM` e.t.c